### ISSUE TITLE
Possibly wrong post create permission check

### ISSUE THREAD

**Issue by kotcrab** [rr-/szurubooru#90]
I'm trying to upload image via rest API using this code (code here is not *that* important but you should see that request is fine):
```kotlin
val json: JsonObject = jsonObject(
     "safety" to "safe",
     "tags" to jsonArray("auto_tagme")
)

Szurubooru(getLocalDto()).post("posts/")
     .timeout(10 * 1000)
     .data("metadata", json.toString())
     .data("content", "img.jpg", FileInputStream(File("""C:\Users\Kotcrab\Desktop\testimg\img.jpg""")))
     .post()

//...

fun post(requestUrl: String): Connection {
    val request = Jsoup.connect("${config.apiPath}$requestUrl").ignoreContentType(true)
    request.header("Authorization", "Basic " + basicHttpAuth)
    return request;
}
```

This results in `AssertionError` on server:
```
waitress Exception when serving /posts/
Traceback (most recent call last):
  File "/root/szuru/server/python_modules/lib/python3.5/site-packages/waitress/channel.py", line 338, in service
    task.service()
  File "/root/szuru/server/python_modules/lib/python3.5/site-packages/waitress/task.py", line 169, in service
    self.execute()
  File "/root/szuru/server/python_modules/lib/python3.5/site-packages/waitress/task.py", line 399, in execute
    app_iter = self.channel.server.application(env, start_response)
  File "/root/szuru/server/python_modules/lib/python3.5/site-packages/falcon/api.py", line 189, in __call__
    responder(req, resp, **params)
  File "/root/szuru/server/szurubooru/api/base_api.py", line 7, in _wrapper_method
    actual_method(request.context, *args, **kwargs)
  File "/root/szuru/server/szurubooru/api/post_api.py", line 25, in post
    auth.verify_privilege(ctx.user, 'posts:create')
  File "/root/szuru/server/szurubooru/func/auth.py", line 60, in verify_privilege
    if not has_privilege(user, privilege_name):
  File "/root/szuru/server/szurubooru/func/auth.py", line 52, in has_privilege
    assert privilege_name in config.config['privileges']
AssertionError
```

Seems like it looks for `posts:create` in config but it doesn't exist. There is only `posts:create:anonymous` and `posts:create:identified`. 

As a workaround I added `'posts:create': regular` to config and I was able to successfully upload file.

**Comment by rr-** [comment]
Those are leftovers over what I forgot to implement. Which has reminded me to actually implement them. Thanks - I'll fix it in the morning.

**Comment by rr-** [comment]
Fixed in 508cb6e.
